modules/ding_react/plugins/content_types/related_materials.inc

```diff
@@ -83,15 +83,21 @@ function ding_react_related_materials_content_type_render($subtype, $conf, $pane
 /**
  * Format a set of string values to a format suitable for React app data.
  *
- * @param string[] $strings
- *   The string values.
+ * @param mixed $data
+ *   The data to format. Either string or array.
```

Comment:
If we are reacting to the signature of [`getAudience()`](https://github.com/ding2/ding2/blob/df300c08f29d4d05b0207c3304959ecc51cd4943/modules/ting/src/TingObjectInterface.php#L101-L109) and other functions I think it would be reasonable to document the types accepted to be `string|string[]|FALSE` instead of `mixed`.